code/game/gamemodes/objective.dm

```diff
@@ -421,6 +421,55 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 			return ..()
 	return SSshuttle.emergency.is_hijacked() || ..()
 
+/datum/objective/gimmick
+	name = "gimmick"
+	martyr_compatible = TRUE
+
+/datum/objective/gimmick/update_explanation_text()
+	var/selected_department = pick(list( //Select a department for department-based objectives
+		DEPT_SCIENCE,
+		DEPT_ENGINEERING,
+		DEPT_SECURITY,
+		DEPT_MEDICAL,
+		DEPT_SERVICE,
+		DEPT_SUPPLY,
+		DEPT_COMMAND
+	))
+	var/list/gimmick_list = list( //This list is ONLY for objectives that depend on a random selection
+		"Get everyone to hate [selected_department], through misdirection, slander, and sabotage.",
+		"Incite a strike in [selected_department].",
+		"You are a bully. Be a real jerk to those nerds at [selected_department] whenever possible. NEEERDS!",
```

Comment:
perhaps use a slightly different flavor here

Reply:
changed to "Impede the work of those at [selected_department] while keeping those working in it alive."